### PR TITLE
Support for static url

### PR DESCRIPTION
**Pull request by marianoeramirez**
I want to merge this change because. I think is important to have the ability to edit the location of the assets with the STATIC_URL environment parameter like the dashboard application.

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

**Comment by orzechdev** [comment]
Hi, thank you for contributing! It is definitely important. May I ask you for a rebase with master to resolve conflicts? We will then see that all tests pass and we will be able to merge changes.

**Comment by bomba1990** [comment]
@orzechdev I merge and resolved the conflicts, The only test that is not passing is the build because it requires the API_URL. But not sure how to solve this. Do you have any suggestion?

**Comment by orzechdev** [comment]
@bomba1990 Oh, great! We do not have configured CircleCI to run tests with forked repos, but I have run it locally and it passes all of them. Unfortunately, I see there is unnecessarily duplicated `devtool: "source-map",`  and also `output` property, which should be removed in the `config/webpack/config.base.js`. May you fix it too?

**Comment by bomba1990** [comment]
@orzechdev Done, I Fixed the reported by LGTM.

File: config/webpack/config.base.js [hunk 1]
```diff
@@ -42,7 +43,7 @@ module.exports = ({ sourceDir, distDir }) => ({
             options: {
               name: "[name].[ext]",
               outputPath: "fonts/",
-              publicPath: "/fonts/",
+              publicPath: STATIC_URL+"fonts/",
```
Comment (dominik-zeglen) [inline config/webpack/config.base.js:46]: You don't have to concatenate strings here - it's sufficient to just put `/fonts/` here

File: config/webpack/config.base.js [hunk 2]
```diff
@@ -55,7 +56,7 @@ module.exports = ({ sourceDir, distDir }) => ({
             options: {
               name: "[name].[ext]",
               outputPath: "images/",
-              publicPath: "/images/",
+              publicPath: STATIC_URL+"images/",
```
Comment (dominik-zeglen) [inline config/webpack/config.base.js:59]: Same here

File: config/webpack/config.base.js [hunk 3]
```diff
@@ -84,7 +85,7 @@ module.exports = ({ sourceDir, distDir }) => ({
   },
   output: {
     path: distDir,
-    publicPath: "/",
+    publicPath: STATIC_URL,
```
Comment (dominik-zeglen) [inline config/webpack/config.base.js:88]: This line adds public paths so you don't have to do this for every asset (images, fonts, etc)

File: config/webpack/config.base.js (hunk 1 above)
Comment (bomba1990) [reply config/webpack/config.base.js]: ok, I solved this on my branch.

File: config/webpack/config.base.js (hunk 2 above)
Comment (bomba1990) [reply config/webpack/config.base.js]: ok, I solved this on my branch.

**Comment by bomba1990** [comment]
@dominik-zeglen I reviewed the requested, let me now if left any other comment.

**Comment by marianoeramirez** [comment]
@dominik-zeglen @orzechdev Do we have any update on this?

**Pull request merged**